backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -186,6 +186,7 @@ private CloneImageGroupVolumesStructureCommandParameters buildCloneImageGroupVol
         params.setDestImageGroupId(getImageGroupId());
```

Comment:
Starting with a commit message will be helpful for reviews, to understand what you are trying to do.

Reply:
Sorry Nir, I didn't understand this comment :(
There is a commit message, should I improve it?

Reply:
There commit message is:

    WIP

This means that this commit is not ready for review.

How to improve it:
- Explain what you are trying to do
- Why you chose the current implementation
- Explain why this is work in progress - what is missing, known issues

Reply:
OK, OK, I see now.
You left this comment in a commit that actually has a commit message, that what confused me :)
Now I understand that you are talking about last commit in the chain, thanks!